website/content/partials/raft-panic-old-tls-key.mdx

```diff
@@ -0,0 +1,17 @@
+## Integrated Storage panic related to old TLS key
+
+Raft in Vault uses its own set of TLS certificates, independent of those that the user
```

Comment:
The whole issue writeup is really good - solid background and description of when it can occur.